topics/discord-bots/index.md

```diff
@@ -0,0 +1,10 @@
+---
+aliases: discordbots
+topic: discord-bots
+github_url: https://github.com/discord
+display_name: Discord Bots (Extensions)
+logo: discord-bots.png
+short_description: Discord Bots is extensions for Discord chat app. 
+url: https://top-bots.xyz
+---
+A bot is an automated Discord account. It utilises the Discord API. They have a "BOT" tag next to their username. They can be added through the API. Bots typically follow a command structure, where a user sends a prefixed message and the bot reponds, though bots can work in many different ways.
```

Comment:
```suggestion
A bot is an automated Discord account. It utilizes the Discord API. They have a "BOT" tag next to their username. They can be added through the API. Bots typically follow a command structure, where a user sends a prefixed message or a slash command, and the bot reponds, though bots can work in many different ways.
```
I think it would be more consistent to use the American spelling of things (like the words in other pages, e.g. "color"). Second, slash commands are also dominant in today's era of Discord [bots]. Third, of course, the Oxford comma before the "and" 😄.

Reply:
Thank you for correcting me! <3